source/includes/kubernetes/_k8_deployments.md

```diff
@@ -61,34 +61,35 @@ curl -X GET \
 
 ```json
 {
-    "data": {
-      "id": "apm-server-1585600296-apm-server/monitoring",
-      "deploymentStatus": "Available",
-      "readyRatio": "1/1",
-      "metadata": {},
```

Comment:
I know this isn't related to this PR necessarily but should we be including examples of the nested fields of the metadata, spec in the example response. These are typically never empty

Reply:
The problem with adding example would be the API docs would consist of mostly json objects. We refer them to the Kubernetes API docs instead.